lib/forkcms_deploy/forkcms_3.7.rb

```diff
@@ -1,9 +1,7 @@
-require "forkcms_deploy/forkcms_3"
+require "forkcms_deploy/forkcms_3.5"
 configuration = Capistrano::Configuration.respond_to?(:instance) ? Capistrano::Configuration.instance(:must_exist) : Capistrano.configuration(:must_exist)
 
 configuration.load do
-	set :shared_children, %w(files config install)
-
 	after 'deploy:update_code' do
 		composer.install_vendors
```

Comment:
tasks composer:install_vendors and composer:install_composer are also in forkcms_3.4.rb